caffe2/operators/quantized/int8_simd.h

```diff
@@ -19,10 +19,11 @@
 
 #if defined(__SSE4_2__) && defined(__clang__)
 #define INT8_NEON_SIMD
+#include "NEON_2_SSE.h"
```

Comment:
I don't see the neon2sse.h file in pytorch repo. Instead, we have ./third_party/neon2sse/NEON_2_SSE.h in repo.

Reply:
I am puzzled as to how it worked before